service/service.go

```diff
@@ -87,7 +71,7 @@ func Create(
 	// For non-whitelisted endpoints, do identityHandler or corsHandler
 	if cfg.IsPublishing {
 		log.Event(ctx, "private endpoints are enabled. using identity middleware")
-		identityHandler := identity.Handler(cfg.ZebedeeURL)
+		identityHandler := identity.HandlerForHTTPClient(clientsidentity.NewAPIClient(zc.Client, cfg.ZebedeeURL))
```

Comment:
We might want to use dp-api-clients-go identity here, instead of go-ns/identity

Reply:
Should we be doing this as part of this PR?

Reply:
I think this should be part of another issue.
In this task I created the healthcheck client here (using NewAPIClient from dp-api-clients-go), and used it in the go-ns identity Handler. Before this change, we were creating the HTTP client in go-ns/identity

I imagine at some point we want to move the Handler out of go-ns/identity into dp-api-clients-go/identity, but I think this should be a separate task. What do you think?